qiskit/circuit/quantumcircuit.py

```diff
@@ -265,7 +257,7 @@ def cast(value, _type):
     @staticmethod
     def _bit_argument_conversion(bit_representation, in_array):
         try:
-            if _is_bit(bit_representation):
+            if isinstance(bit_representation, Bit):
```

Comment:
We're dropping support here for specifying bits as tuples. Do we not want this around for backwards compatibility? If not, does it warrant a deprecation warning?

Reply:
Fixed in fc4e0ea9